src/kvstore/NebulaStore.cpp

```diff
@@ -359,14 +364,31 @@ ErrorOr<nebula::cpp2::ErrorCode, HostAddr> NebulaStore::partLeader(GraphSpaceID
 void NebulaStore::addSpace(GraphSpaceID spaceId, bool isListener) {
   folly::RWSpinLock::WriteHolder wh(&lock_);
   if (!isListener) {
+    // Iterate over all engines to ensure that each dataPath has an engine
```

Comment:
It's better to make it clear in the comments that disk paths may not have kv engines created after recovering from repair.